apps/block_scout_web/lib/block_scout_web/controllers/address_celo_controller.ex

```diff
@@ -16,6 +17,7 @@ defmodule BlockScoutWeb.AddressCeloController do
          {:ok, address} <- Chain.hash_to_address(address_hash),
          %CeloAccount{address: _} <- address.celo_account do
       Logger.debug("Parsing Celo Address #{address_hash_string}")
+      AddressContractVerificationController.check_sourcify(address_hash_string, conn)
```

Comment:
I guess we don't need to do that on every tab, so I'd suggest to drop it here...